product_docs/docs/biganimal/release/getting_started/creating_a_cluster/creating_a_dha_cluster.mdx

```diff
@@ -54,30 +55,30 @@ The following options aren't available when creating your cluster:
 
 1.  Under **Tags**, select **+**.
 
-1. To assign an existing tag, in the search bar under **Tags**, enter a tag name. To add a new tag, instead select **+ Add Tag**.
+1.  To assign an existing tag, in the search bar under **Tags**, enter a tag name. To add a new tag, instead select **+ Add Tag**.
 
 1.  Select **Next: Data Groups**.
 
 ## Data Groups Tab
 
-1. Go to the **Nodes Settings** tab for the first Data Group. In the **Nodes** section, select **Two Data Nodes** or **Three Data Nodes**.  
+1.  Go to the **Node Settings** tab for the first Data Group. In the **Nodes** section, select **Two Data Nodes** or **Three Data Nodes**.  
 
      For more information on node architecture, see [Distributed high availability](../../overview/02_high_availability/distributed_highavailability/).
 
-1.  In the **Provider & Region** section, select the provider and region where you want to deploy your cluster.
+1.  In the **Provider & Region** section, select the cloud provider and region where you want to deploy your cluster. If you're using your account and haven't connected it to BigAnimal yet, see [Connecting to your cloud](/biganimal/latest/getting_started/02_connecting_to_your_cloud)
```

Comment:
Note to self: This is too late here, either put it on the top at the cluster config section, or delete because we have a note at the beginning anyway.